micro_ros_common_diagnostics/src/hwmonitor.c

```diff
@@ -70,43 +68,44 @@ int main(int argc, const char * argv[])
 
   // updater
   diagnostic_updater_t updater;
-  rc = rclc_diagnostic_updater_init(
-    &updater,
-    &my_node,
-    UPDATER_NAME,
-    UPDATER_DESC,
-    UPDATER_HW_ID);
+  rc = rclc_diagnostic_updater_init(&updater, &updater_node, 17, 42);
   if (rc != RCL_RET_OK) {
     printf("Error in creating diagnostic updater\n");
     return -1;
   }
-
   diagnostic_task_t task;
   rc = rclc_diagnostic_task_init(
-    &task,
-    TASK_NAME,
-    &my_diagnostic_function);
+    &task, 0,
```

Comment:
Maybe 0 should be a predefined `id`?

Reply:
Now a command line argument, so common diagnostics can be deployed more flexibly.